examples/next-openai/components/tool/anthropic-code-execution-view.tsx

```diff
@@ -159,7 +197,31 @@ function InputView({
     ReturnType<typeof anthropic.tools.codeExecution_20250825>
   >['input'];
 }) {
-  switch (input?.type) {
+  if (!input) {
+    return null;
+  }
+
+  // Handle programmatic tool calling format: { code: string }
+  if ('code' in input && !('type' in input)) {
```

Comment:
this is strange. can we instead inject a fake type 'programmatic-tool-call' in our mapping?